azure-pipelines.yml

```diff
@@ -31,6 +31,22 @@ stages:
         ptf_name: ptf_vms6-1
         tbtype: t0
         vmtype: ceos
+        section: part-1
+
+  - job:
+    pool: sonictest
+    displayName: "kvmtest-t0-part2"
+    timeoutInMinutes: 400
```

Comment:
Probably we can consider reducing this timeout value now.

Reply:
Good point, update it to 200.